lib/realtime/tenants/connect.ex

```diff
@@ -97,37 +95,21 @@ defmodule Realtime.Tenants.Connect do
     Logger.metadata(external_id: tenant_id, project: tenant_id)
 
     with %Tenant{} = tenant <- Tenants.get_tenant_by_external_id(tenant_id),
-         res <- Database.check_tenant_connection(tenant, @application_name),
+         {:ok, conn} <- Database.check_tenant_connection(tenant, @application_name),
+         ref = Process.monitor(conn),
```

Comment:
Is it worth using link() instead of monitor()?

Reply:
We're using monitor here to ensure that we track the connection shutdown and handle it to also kill the Connect process